### PR TITLE
Feature/input dictionary minrank estimator

### PR DESCRIPTION
**Pull request by Javierverbel**
### Description
- Input dictionary minrank estimator

### Review process
- Verify that FrontEnd displays correctly.

### CODE REVIEW

File: input_dictionary.json [hunk 1]
```diff
@@ -750,7 +803,130 @@
...
...tps://eprint.iacr.org/2022/1226.pdf)\n\n[[KS99]](https://link.springer.com/chapter/10.1007/3-540-48405-1_2) Aviad Kipnis and Adi Shamir. Cryptanalysis of the HFE Public Key Cryptosystem by Relinearization.\n\n[[BB22]](https://link.springer.com/chapter/10.1007/978-3-031-17234-2_6) Author name. Improvement of Algebraic Attacks for Solving Superdetermined MinRank Instances. [[eprint]](https://arxiv.org/abs/2208.01442)\n\n[[FSS10]](https://members.loria.fr/PJSpaenlehauer/data/papers/FauSafSpa10a.pdf) Jean-Charles Faugère , Mohab Safey El Din, and Pierre-Jean Spaenlehauer. Computing Loci of Rank Defects of Linear Matrices using Gröbner Bases and Applications to Cryptology.\n\n[[FSS13]](https://www.sciencedirect.com/science/article/pii/S0747717113000485) Author name. On the Complexity of the Generalized MinRank Problem. [[eprint]](https://arxiv.org/abs/1112.4411)\n\n[[GNS23]](https://dl.acm.org/doi/fullHtml/10.1145/3597066.3597077) Sriram Gopalakrishnan, Vincent Neiger, and Mohab Safey El Din. Refined F5 Algorithms for Ideals of Minors of Square Matrices. [[eprint]](https://arxiv.org/abs/2302.05375#:~:text=We%20consider%20the%20problem%20of,characteristic%20zero%20or%20large%20enough.)",
```
Comment (FloydZ) [inline input_dictionary.json:828]: For the following paper are no authors listed:
- On the Complexity of the Generalized MinRank Problem: Jean-Charles Faugère, Mohab Safey El Din, Pierre-Jean Spaenlehauer
- Improvement of algebraic attacks for solving superdetermined MinRank instances: Magali Bardet, Manon Bertin

Comment (FloydZ) [reply input_dictionary.json:828]: There is only a template: "Authors name."

**Review by FloydZ — CHANGES_REQUESTED** [review]

File: input_dictionary.json (hunk 1 above)
Comment (Javierverbel) [reply input_dictionary.json]: Done

**Review by FloydZ — APPROVED** [review]

**Comment by FloydZ** [comment]
can be merged

**Review by FloydZ — APPROVED** [review]

**Pull request merged**